understanding/21/reflow.html

```diff
@@ -12,68 +12,245 @@ <h1>Understanding Reflow</h1>
-      </section>
-      <section>
-      <h3>Visibility and availability of content</h3> 
-      <p>How much of the content is visible may change at different scales. For example, navigation menus that are fully visible in the desktop layout are often collapsed into fewer items, or even into a single menu button (the 'hamburger' icon pattern) so they take up less screen space.</p>
-      <p>The Success Criterion is met as long as all content and functionality are still fully available -  either directly, or revealed via accessible controls, or accessible via direct links.</p>
-      </section>
+        <p>The intent of this success criterion is to let users enlarge (zoom in) text and other content without having to perform bi-directional scrolling to read. Lines of text that extend beyond the viewport in two-directions force users to scroll back-and-forth to read, which can cause them to lose their place, and significantly increase both physical and cognitive effort. Sections of content are expected to reflow within the appropriate sizing requirement defined by this success criterion, in regards to the text's intended direction of reading. With exceptions for sections of content which functionally requires bi-directional layout for usage or meaning.</p>
+
+        <p>Reflow applies to both horizontally and vertically written languages. For pages where the primary written language expects vertical scrolling (such as English), users do not expect to scroll back-and-forth horizontally to read that content. Similarly, users reading content in a written language/direction that expects horizontal scrolling (such as traditional Chinese and Japanese) do not expect to scroll up-and-down vertically to read that content.</p>
+
+        <div class="note">
+          <p>Reflow does not prohibit web pages from presenting both horizontal and vertical scrollbars for individual sections of content. Nor does it disallow the use of bi-directional scrollbars at the page (viewport) level if the scrollbars are necessary to scroll excepted content, and non-excepted sections of content otherwise meet Reflow.</p>
+          
+          <p>However, whenever possible it is in the best interest of the user, to reduce physical and cognitive effort, to limit bi-directional scrolling only to the individual sections of content which necessitate such scrolling, rather than allow the page at large to scroll in two directions. For instance, one might maintain a single scrollbar at page level and confine content which needs bi-directional scrolling (such as a table) into a scrollable container.</p>
```

Comment:
```suggestion
          <p>However, whenever possible it is in the best interest of the user to limit bi-directional scrolling only to the individual sections of content which necessitate such scrolling, rather than allowing the page at large to scroll in two directions. For instance, one might maintain a single scrollbar at page level and confine content which needs bi-directional scrolling (such as a table) into a scrollable container.</p>
```